.github/workflows/terragrunt-plan-production.yml

```diff
@@ -24,6 +24,9 @@ env:
   TF_VAR_rds_db_password: ${{ secrets.PRODUCTION_DB_PASSWORD }}
   TF_VAR_slack_webhook: ${{ secrets.PRODUCTION_SLACK_WEBHOOK }}
   TF_VAR_gc_notify_callback_bearer_token: ${{ secrets.PRODUCTION_GC_NOTIFY_CALLBACK_BEARER_TOKEN }}
+  TF_VAR_gc_notify_url: https://notification.canada.ca
```

Comment:
This is not the correct value. It should be https://api.notification.canada.ca . I would recommend hard coding this value since its not changing